### ISSUE TITLE
keyboard and mouse system check fails even after enabling them in accessibility panels

### ISSUE THREAD

**Issue by joshryu0425** [justingardner/mgl#74]
mglTestKeys pass on terminal, but not on matlab gui

**Comment by justingardner** [comment]
Seems to be working for me now, both on terminal and in Matlab GUI. Closing this out.
